ansible-ipi-install/roles/node-prep/tasks/70_enabled_fw_services.yml

```diff
@@ -1,19 +1,52 @@
 ---
-- name: Enable Firewall for HTTP
-  firewalld:
-    service: http
-    permanent: yes
-    state: enabled
-  become: yes
+- name: Configure firewalld
+  block:
+    - name: Enable Firewall for HTTP
+      firewalld:
+        service: http
+        permanent: yes
+        state: enabled
+      become: yes
 
-- name: Open Firewall port 5000/tcp, zone libvirt and public
-  firewalld:
-    port: 5000/tcp
-    permanent: yes
-    state: enabled
-    zone: "{{ item }}"
-  become: yes
-  with_items:
-  - libvirt
-  - public
+    - name: Open Firewall port 5000/tcp, zone libvirt and public
+      firewalld:
+        port: 5000/tcp
+        permanent: yes
+        state: enabled
+        zone: "{{ item }}"
+      become: yes
+      with_items:
+      - libvirt
+      - public
+  when: firewall != "iptables"
 
+- name: Configure iptables
+  block:
+    - name: Enable Firewall for HTTP
+      iptables:
+        chain: INPUT
+        protocol: tcp
+        destination_port: "80"
+        jump: ACCEPT
+      become: yes
+
+    - name: Open Firewall port 5000/tcp
+      iptables:
+        chain: INPUT
+        protocol: tcp
+        destination_port: "5000"
+        jump: ACCEPT
+      become: yes
+
+    - name: Allow related and established connections
+      iptables:
+        chain: INPUT
+        ctstate: ESTABLISHED,RELATED
+        jump: ACCEPT
+      become: yes
+
+    - name: Save iptables configuration
+      shell: |
+        /usr/sbin/iptables-save > /etc/sysconfig/iptables
+      become: yes
```

Comment:
I think you need to check if the iptables service is up otherwise enable it

Reply:
That is handled in [ansible-ipi-install/roles/node-prep/tasks/60_enabled_services.yml](https://github.com/openshift-kni/baremetal-deploy/pull/139/files#diff-3465117d70f6807e3a48d826b378b960)